src/Components/Onboarding/Steps/Artists/ArtistSearchResults.tsx

```diff
@@ -25,14 +25,15 @@ class ArtistSearchResultsContent extends React.Component<RelayProps, null> {
   }
 
   onArtistFollowed(artistId: string, store: RecordSourceSelectorProxy, data: SelectorData): void {
-    const suggestedArtist = store.get(data.followArtist.artist.related.suggested.edges[0].node.__id)
-    this.excludedArtistIds.add(suggestedArtist.getValue("_id"))
+    const suggestedArtistNode = data.followArtist.artist.related.suggested.edges[0]
+    const popularArtist = data.followArtist.popular_artists.artists[0]
+    const artistToSuggest = store.get(((suggestedArtistNode && suggestedArtistNode.node) || popularArtist).__id)
```

Comment:
I wish this could've been cleaner, but wasn't sure any other ways.

Reply:
makes sense to me!

Reply:
Perhaps some spread trickery like this picks your fancy?

```
$ node
> let suggestedArtists = []
undefined

> const popularArtists = ['Banksy']
undefined
> [...suggestedArtists, ...popularArtists]
[ 'Banksy' ]
> [...suggestedArtists, ...popularArtists][0]
'Banksy'

> suggestedArtists = [{ node: 'Vincent Van Gogh' }].map(edge => edge.node)
[ 'Vincent Van Gogh' ]
> [...suggestedArtists, ...popularArtists]
[ 'Vincent Van Gogh', 'Banksy' ]
> [...suggestedArtists, ...popularArtists][0]
'Vincent Van Gogh'
```